blueprints/gke/autopilot/variables.tf

```diff
@@ -81,4 +74,17 @@ variable "region" {
   description = "Region."
   type        = string
   default     = "europe-west1"
+}
+
+variable "vpc_create" {
```

Comment:
I'd merge `vpc_create` and `vpc_name`, maybe even use the existing `cluster_network_config`